runtime/lua/vim/_defaults.lua

```diff
@@ -67,6 +67,21 @@ do
   --- See |&-default|
   vim.keymap.set('n', '&', ':&&<CR>', { desc = ':help &-default' })
 
+  --- Use Q in visual mode to execute a macro on each line of the selection. #21422
+  ---
+  --- Applies to @x and includes @@ too.
+  vim.keymap.set(
+    'v',
+    'Q',
+    ':normal @<c-r>=reg_recorded()<CR><CR>',
+    { silent = true, desc = ':help Q' }
+  )
+  vim.keymap.set(
+    'v',
+    '@',
+    "':normal @'.getcharstr().'<CR>'",
```

Comment:
Should probably be `:normal!`, unless there's a reason otherwise?

```suggestion
    ':normal! @<c-r>=reg_recorded()<CR><CR>',
    { silent = true, desc = ':help Q' }
  )
  vim.keymap.set(
    'v',
    '@',
    "':normal! @'.getcharstr().'<CR>'",
```

Reply:
Agree, on it.